### ISSUE TITLE
Doctypes without a template

### ISSUE THREAD

**Issue by tompipe** [umco/umbraco-doc-type-grid-editor#53]
Grid preview doesn't work on doctypes without a template set

**Comment by tompipe** [comment]
Related to this issue I believe, is when you don't have any nodes published and you get this screen at the front end....

![image](https://cloud.githubusercontent.com/assets/629719/19930788/e497f12a-a101-11e6-9264-1ff128ed74d2.png)

When editing datatypes in the back end, it tries to render a preview....

![image](https://cloud.githubusercontent.com/assets/629719/19930865/3416d770-a102-11e6-81b5-3aeb9ae27225.png)

and pulls in the "welcome" content instead, this is injecting styles and scripts into the backoffice, and causing a huge layout issue!

![image](https://cloud.githubusercontent.com/assets/629719/19930937/911e11e0-a102-11e6-8051-b45b3d42ce44.png)

**Comment by mattbrailsford** [comment]
Yup, this is a known limitation, the preview code for DTGE uses the built in alt templates feature. It'll  try and rung against the current node, or the home node if that's not published. If neither are published, all hell breaks loose ;)

**Comment by tompipe** [comment]
LOL - and now I managed to get a doctype allowed at root (all the buttons are hidden, thank f**k for the keyboard shortcuts)

It's rendering that welcome page in ALL my grid editors!

![image](https://cloud.githubusercontent.com/assets/629719/19931194/80f13c42-a103-11e6-946e-fa55d2a17b04.png)

**Comment by leekelleher** [comment]
@tompipe Just checking, are any of the nodes published? I'm guessing not.

As @mattbrailsford says, it's a known limitation of DTGE's preview feature... as always, we're open for suggestions.

**Comment by tompipe** [comment]
No, no nodes published at all. It's UaaS and have just done a sync between environments, but it wont sync content because doctypes differ, so was going to manually change them to match, when "all hell broke loose" :D 

I'm working again now, don't have time to look at a fix, but I will have a look if/when I get time. Just wanted to raise it in case.

**Comment by leekelleher** [comment]
Closing this specific ticket.  The issue still remains, but want to combine it with ticket #59 - for an overhaul of the preview mechanism.
